packages/core/src/auth/ui/vue/show.ts

```diff
@@ -718,7 +718,7 @@ export function getShowManageConnections(): RegisteredCommand<any> {
 export function registerCommands(context: vscode.ExtensionContext, prefix: string) {
     showManageConnections = Commands.register(
         { id: `aws.${prefix}.auth.manageConnections`, compositeKey: { 1: 'source' } },
-        (_: VsCodeCommandArg, source: AuthSource, serviceToShow?: ServiceItemId) => {
+        async (_: VsCodeCommandArg, source: AuthSource, serviceToShow?: ServiceItemId) => {
```

Comment:
you can add await to where this command is executed

Reply:
command executions are always async, so no change to callers.